controllers/storagecluster/generate.go

```diff
@@ -42,6 +42,17 @@ func generateNameForCephBlockPool(initData *ocsv1.StorageCluster) string {
 	return fmt.Sprintf("%s-cephblockpool", initData.Name)
 }
 
+func generateDeviceClass(initData *ocsv1.StorageCluster) string {
+	if initData.Spec.Replica1 != nil && initData.Spec.Replica1.Enable {
+		return "replicated"
```

Comment:
`"replicated"` is used in several places, how about defining a const?

Reply:
I only see the string used twice, so I don't think that's worth defining a const. That said, does Rook have any defined values for device classes?